test/ssc_test/cmod_battery_pvsamv1_test.cpp

```diff
@@ -303,13 +303,13 @@ TEST_F(CMPvsamv1BatteryIntegration_cmod_pvsamv1, ResidentialDCBatteryModelIntegr
 	set_array(data, "load", load_profile_path, 8760); // Load is required for peak shaving controllers
 
 	ssc_number_t expectedEnergy[3] = { 8634, 8637, 8703 };
-	ssc_number_t expectedBatteryChargeEnergy[3] = { 838.3, 807.2, 528.65 };
-	ssc_number_t expectedBatteryDischargeEnergy[3] = { 764.4, 735.4, 484.5 };
+	ssc_number_t expectedBatteryChargeEnergy[3] = { 1412.75, 1414.89, 253.2 };
+	ssc_number_t expectedBatteryDischargeEnergy[3] = { 1283.8, 1285.88, 226.3 };
```

Comment:
Looks like the PV forecast made quite a difference all the pv + dc battery results! Could you please add a test seeing what happens if you take the actual dc clipped power loss and use that as the clipping prediction, to make sure that the results are the same regardless of if there is a dc clipped power forecast or not?

Reply:
Great, thanks!